src/GraphQL/Types/Composite/AutoRegisteringObjectGraphType.cs

```diff
@@ -121,15 +123,27 @@ protected void BuildFieldType(FieldType fieldType, MemberInfo memberInfo)
                         queryArgument ?? throw new InvalidOperationException("Invalid response from ConstructQueryArgument: queryArgument and expression cannot both be null"));
                     expressions.Add(expression);
                 }
-                var resolver = new MemberResolver(methodInfo, BuildMemberInstanceExpression(memberInfo), expressions);
+                var memberInstanceExpression = BuildMemberInstanceExpression(methodInfo);
+                if (IsObservable(methodInfo.ReturnType))
+                {
+                    var resolver = new EventStreamMethodResolver(methodInfo, memberInstanceExpression, expressions);
+                    fieldType.Resolver = resolver;
```

Comment:
Both branches contain this line.

Reply:
Yeah.... I'd have to write `IFieldResolver resolver` to move it outside the `if`.  Or I can consolidate the line with the line above it in each branch.  Seemed best to leave it this way.